lib/cmap/edges_to_queries.rb

```diff
@@ -35,7 +38,10 @@ def gsub_subqueries
   end
 
   def updates
-    u = (grouped_edges[false] || []).map {|e| "#{e.destination_vertex}=(#{e.value})::int"}.join(", ")
+    u = (grouped_edges[false] || []).map do |e|
+      cp = ColumnParser.new(e.destination_vertex)
+      "#{cp.column}=(#{e.value})::int"
```

Comment:
I'm not sure we want to use a different vertex name here.  I think we'll need to make it so `int` isn't hardcoded anymore.